src/installer/publish/publish-final.proj

```diff
@@ -28,8 +28,8 @@
   <Target Name="FinalizeBuildInAzure"
           DependsOnTargets="
             GetProductVersions;
-            GetLatestCommitHash">
-    <Error Condition="'$(LatestCommit)' == ''" Text="Missing required property 'LatestCommit'" />
+            InitializeSourceControlInformationFromSourceControlManager">
+    <SourceRevisionId Condition="'$(SourceRevisionId)' == ''">N/A</SourceRevisionId>
```

Comment:
In the absence of source control, the value of `SourceRevisiionId` is empty. Fallback value `N/A` to make it a drop-in replacement of `GetLatestCommitHash` target being deleted.